plugins/catalog-backend-module-bitbucket-server/src/providers/BitbucketServerEntityProvider.ts

```diff
@@ -207,10 +231,363 @@ export class BitbucketServerEntityProvider implements EntityProvider {
             presence: 'optional',
           },
         })) {
+          entity.metadata.annotations![this.defaultBranchAnnotation] =
```

Comment:
I will check on this.

Reply:
@camilaibs The default parser adds the annotation to the location entity, and then I simply add to this. So someone were to change the parser, then perhaps this would cause an error. To guarantee this doesn't cause an issue I can just check for this and add it if necessary at this stage.